diff --git a/Makefile b/Makefile
index bc2625c..9793998 100644
--- a/Makefile
+++ b/Makefile
@@ -33,7 +33,8 @@
 
 ARCH=i386
 #ARCH=x86_64
-CC=gcc34
+#CC=gcc34
+CC=gcc
 BASE_CFLAGS=-Dstricmp=strcasecmp -D_strnicmp=strncasecm -Dstrnicmp=strncasecmp -Dstrcmpi=strcasecmp
 
 #use these cflags to optimize it
diff --git a/g_items.c b/g_items.c
index 8a9c4d7..ee661f7 100644
--- a/g_items.c
+++ b/g_items.c
@@ -24,11 +24,11 @@ gitem_armor_t jacketarmor_info	= { 25,  50, .30, .00, ARMOR_JACKET};
 gitem_armor_t combatarmor_info	= { 50, 100, .60, .30, ARMOR_COMBAT};
 gitem_armor_t bodyarmor_info	= {100, 200, .80, .60, ARMOR_BODY};
 
-static int	jacket_armor_index;
-static int	combat_armor_index;
-static int	body_armor_index;
-static int	power_screen_index;
-static int	power_shield_index;
+int	jacket_armor_index;
+int	combat_armor_index;
+int	body_armor_index;
+int	power_screen_index;
+int	power_shield_index;
 
 #define HEALTH_IGNORE_MAX	1
 #define HEALTH_TIMED		2
diff --git a/p_weapon.c b/p_weapon.c
index bc2c487..78fd65f 100644
--- a/p_weapon.c
+++ b/p_weapon.c
@@ -5,7 +5,7 @@
 #include "admin.h"
 
 
-static qboolean	is_quad;
+qboolean	is_quad;
 static byte		is_silenced;
 
 
