diff --git a/src/server/game.c b/src/server/game.c
index 6a335f0f..adee2c23 100644
--- a/src/server/game.c
+++ b/src/server/game.c
@@ -101,7 +101,7 @@ static void PF_Unicast(edict_t *ent, qboolean reliable)
 
     client = svs.client_pool + clientNum;
     if (client->state <= cs_zombie) {
-        Com_WPrintf("%s to a free/zombie client %d\n", __func__, clientNum);
+        //Com_WPrintf("%s to a free/zombie client %d\n", __func__, clientNum);
         goto clear;
     }
 
@@ -251,7 +251,7 @@ static void PF_cprintf(edict_t *ent, int level, const char *fmt, ...)
 
     client = svs.client_pool + clientNum;
     if (client->state <= cs_zombie) {
-        Com_WPrintf("%s to a free/zombie client %d\n", __func__, clientNum);
+        //Com_WPrintf("%s to a free/zombie client %d\n", __func__, clientNum);
         return;
     }
 
