diff -u original/g_items.c modified/g_items.c
--- original/g_items.c	2024-12-09 16:43:02.149831727 -0800
+++ modified/g_items.c	2024-12-09 16:40:18.035198098 -0800
@@ -34,9 +34,9 @@
 gitem_armor_t combatarmor_info	= { 50, 100, .60, .30, ARMOR_COMBAT};
 gitem_armor_t bodyarmor_info	= {100, 200, .80, .60, ARMOR_BODY};
 
-static int	jacket_armor_index;
-static int	combat_armor_index;
-static int	body_armor_index;
+int	jacket_armor_index;
+int	combat_armor_index;
+int	body_armor_index;
 static int	power_screen_index;
 static int	power_shield_index;
 
diff -u original/g_local.h modified/g_local.h
--- original/g_local.h	2024-12-09 16:43:02.149831727 -0800
+++ modified/g_local.h	2024-12-09 16:40:18.035198098 -0800
@@ -22,13 +22,13 @@
 //==================================================================
 
 #ifndef _WIN32
-#include <nan.h>
+#include <math.h>
 #define min(a,b) ((a) < (b) ? (a) : (b))
 #define max(a,b) ((a) > (b) ? (a) : (b))
 #ifdef __sun__
 #define _isnan(a) (NaN((a)))
 #else
-#define _isnan(a) ((a)==NAN)
+#define _isnan(a) (isnan(a))
 #endif
 #endif
 
diff -u original/g_phys.c modified/g_phys.c
--- original/g_phys.c	2024-12-09 16:43:02.149831727 -0800
+++ modified/g_phys.c	2024-12-09 16:40:18.036198102 -0800
@@ -962,6 +962,7 @@
 		case MOVETYPE_STOP:
 			SV_Physics_Pusher (ent);
 			break;
+        case MOVETYPE_WALK:
 		case MOVETYPE_NONE:
 			SV_Physics_None (ent);
 			break;
diff -u original/m_boss31.c modified/m_boss31.c
--- original/m_boss31.c	2024-12-09 16:43:02.149831727 -0800
+++ modified/m_boss31.c	2024-12-09 16:40:57.501349762 -0800
@@ -9,7 +9,7 @@
 #include "g_local.h"
 #include "m_boss31.h"
 
-extern SP_monster_makron (edict_t *self);
+extern void SP_monster_makron (edict_t *self);
 qboolean visible (edict_t *self, edict_t *other);
 
 static int	sound_pain1;
diff -u original/m_flyer.c modified/m_flyer.c
--- original/m_flyer.c	2024-12-09 16:43:02.150831731 -0800
+++ modified/m_flyer.c	2024-12-09 16:41:39.197509997 -0800
@@ -720,7 +720,7 @@
 }
 
 // PMM - kamikaze code .. blow up if blocked	
-int flyer_blocked (edict_t *self, float dist)
+qboolean flyer_blocked (edict_t *self, float dist)
 {
 	vec3_t origin;
 
